tests/workflows/test_snowflake.py

```diff
@@ -0,0 +1,110 @@
+import os
+import uuid
+
+import dask.dataframe as dd
+import pandas as pd
+import pytest
+from dask.distributed import wait
+
+pytest.importorskip("dask_snowflake", reason="Requires dask-snowflake")
+
+from dask_snowflake import read_snowflake, to_snowflake  # noqa: E402
+from snowflake.sqlalchemy import URL  # noqa: E402
+from sqlalchemy import create_engine  # noqa: E402
+
+
+@pytest.fixture(scope="module")
+def connection_kwargs():
+    return {
+        "user": os.environ["SNOWFLAKE_USER"],
+        "password": os.environ["SNOWFLAKE_PASSWORD"],
+        "account": os.environ["SNOWFLAKE_ACCOUNT"],
+        "warehouse": os.environ["SNOWFLAKE_WAREHOUSE"],
+        "role": os.environ.get("SNOWFLAKE_ROLE", "public"),
+        "database": os.environ.get("SNOWFLAKE_DATABASE") or "testdb",
+        "schema": os.environ.get("SNOWFLAKE_SCHEMA") or "testschema",
+    }
+
+
+@pytest.fixture
+def table(connection_kwargs):
+    """Connect to snowflake and create table"""
+    name = f"citibike_tripdata_{uuid.uuid4().hex}"
+    engine = create_engine(URL(**connection_kwargs))
+    engine.execute(f"DROP TABLE IF EXISTS {name}")
+    engine.execute(
+        f"""create table if not exists {name} (
+                ride_id varchar not null unique,
+                rideable_type varchar not null,
+                started_at timestamp not null,
+                ended_at timestamp not null,
+                start_station_name varchar not null,
+                start_station_id smallint not null,
+                end_station_name varchar not null,
+                end_station_id smallint not null,
+                start_lat number,
+                start_lng number,
+                end_lat number,
+                end_lng number,
+                is_member boolean not null
+            )"""
+    )
+    yield name
+    # after the data is written, delete table
+    engine.execute(f"DROP TABLE IF EXISTS {name}")
+
+
+@pytest.mark.client("snowflake")
+def test_write(client, connection_kwargs, table):
```

Comment:
nit:
```suggestion
def test_etl_into_snowflake(client, connection_kwargs, table):
```

This test isn't _just_ writing data to Snowflake, it also involves reading a lot of data from csv and a bit of filtering/transforming.